.gitignore

```diff
@@ -15,3 +15,4 @@ crates/oq3_syntax/src/ast/generated/_new_tokens.rs
 .env/
 .envs/
 .ackrc
+save/
```

Comment:
What generates this?

(a bunch of the files in here look like particular files from your directory - you might want to move those to your user-local `.git/info/exclude` rather than having the ignores commited for everyone)

Reply:
Yes that's either cruft or particular things that can be move.